data/google

```diff
@@ -563,7 +560,6 @@ full:google-analytics.com @cn
 full:googleadservices-cn.com @cn
 full:googleadservices.com @cn
 full:googleanalytics.com @cn
-full:googleapis-cn.com @cn
 full:googleapps-cn.com @cn
```

Comment:
If line 566 is unavailable in mainland China, you should remove line 567 simultaneously.